### PR TITLE
Charlesmchen/tweak colors

### PR DESCRIPTION
**Pull request by charlesmchen-signal**
PTAL @michaelkirk

### CODE REVIEW

File: Signal/src/ViewControllers/ConversationView/ConversationStyle.swift [hunk 1]
```diff
@@ -34,7 +34,6 @@ public class ConversationStyle: NSObject {
     @objc public var contentWidth: CGFloat = 0
 
     // viewWidth - (gutterfullWidthGutterLeadingLeading + fullWidthGutterTrailing)
-    // TODO: Is this necessary?
```
Comment (charlesmchen-signal) [inline Signal/src/ViewControllers/ConversationView/ConversationStyle.swift]: I suggest we move this class to SignalMessaging and pull the contents of `OWSMessagesBubbleImageFactory`/`OWSMessagesBubbleColors` into this class.

File: SignalMessaging/categories/UIColor+OWS.h [hunk 2]
```diff
@@ -32,6 +29,37 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (UIColor *)blendWithColor:(UIColor *)otherColor alpha:(CGFloat)alpha;
 
+#pragma mark - New Colors
```
Comment (michaelkirk-signal) [inline SignalMessaging/categories/UIColor+OWS.h]: nit: Nothing is new for long, maybe "Conversation"

File: SignalMessaging/categories/UIColor+OWS.m [hunk 3]
```diff
@@ -180,6 +162,148 @@ - (UIColor *)blendWithColor:(UIColor *)otherColor alpha:(CGFloat)alpha
                            alpha:CGFloatLerp(a0, a1, alpha)];
 }
 
+#pragma mark - New Colors
+
++ (UIColor *)ows_signalBlueColor
+{
+    return [UIColor colorWithRGBHex:0x2090EA];
```
Comment (michaelkirk-signal) [inline SignalMessaging/categories/UIColor+OWS.m]: glad to have this RGBHex method. 👍

File: Signal/src/ViewControllers/ConversationView/ConversationStyle.swift (hunk 1 above)
Comment (michaelkirk-signal) [reply Signal/src/ViewControllers/ConversationView/ConversationStyle.swift]: Sure - is it OK if I include that in my work, or is that blocking something you're doing?

File: SignalMessaging/categories/UIColor+OWS.h (hunk 2 above)
Comment (charlesmchen-signal) [reply SignalMessaging/categories/UIColor+OWS.h]: Removed.  I think we'll be using these colors beyond conversations.

**Pull request closed without merging**